BSB_lan.ino

```diff
@@ -597,6 +597,19 @@ unsigned long custom_timer = millis();
 unsigned long custom_timer_compare = 0;
 float custom_floats[20] = { 0 };
 long custom_longs[20] = { 0 };
+
+#ifdef RGT_EMULATOR
+byte newMinuteValue = 99;
+#endif
+
+#ifdef BUTTONS
+volatile byte PressedButtons = 0;
+#define TWW_PUSH_BUTTON_PRESSED 1
+#define ROOM1_PRESENCE_BUTTON_PRESSED 2
+#define ROOM2_PRESENCE_BUTTON_PRESSED 4
+#define ROOM3_PRESENCE_BUTTON_PRESSED 8
+#endif
+
```

Comment:
I would wrap these lines with #ifndef so that users could define them in _config.h, so that these settings could be defined there and stay the same even if a new version of BSB-LAN is flashed.

Reply:
Or will these be overriden with the EEPROM configuration I just saw below?

Reply:
Thse definitions for internal usage only (just for comfort reading).

Reply:
Ok, great!